source/contribute.rst

```diff
@@ -107,43 +118,40 @@ Primary Branches
 
 The **doctrine** repository holds the following primary branches:
 
--  **doctrine/master** Development towards the next release.
+-  **doctrine/2.7** Development towards the next release.
 -  **doctrine/\*** Maintenance branches of existing releases.
 
 These branches exist in parallel and are defined as follows:
 
-**doctrine/master** is the branch where the source code of **HEAD**
+**doctrine/2.7** is the branch where the source code of **HEAD**
 always reflects the latest version. Each released stable version will be
 a tagged commit in a **doctrine/\*** branch. Each released unstable
-version will be a tagged commit in the **doctrine/master** branch.
+version will be a tagged commit in the **doctrine/2.7** branch.
 
-    **NOTE** You should never commit to your forked **origin/master**.
-    Changes to **origin/master** will never be merged into
-    **doctrine/master**. All work must be done in a **topic branch**,
+    **NOTE** You should never commit to your forked default branch (**origin/2.7**).
```

Comment:
Maybe we should advise people to just remove their local copy of 2.7, that way they will not be tempted to do so.

Reply:
People using the default branch name of the fork is because of convenience. They probably didn't read this docs.